source/includes/fact-mmapv1-big-endian.rst

```diff
@@ -1,5 +1,6 @@
 .. important::
 
-   MMAPv1 is not supported on big-endian architectures such as s390x.
-   MongoDB returns an error if you set MMAPv1 as the
-   storage engine on a big-endian system.
+   :doc:`MMAPv1 </core/mmapv1>` is unsupported on all big-endian
```

Comment:
Rather than trying to state where it isn't supported, why not just state that it is only supported on Intel x86 processors?

Reply:
For instance, you don't state here that it isn't supported on ARM64. The actual logic that determines whether mmapv1 is here:

https://github.com/mongodb/mongo/blob/666545a2c01ad724912391ca44be36981d3ed0d2/SConstruct#L1829-L1836